docs/edge_processor.md

```diff
@@ -2,13 +2,13 @@
 
 ## Intro
 
-`Edge Processor` can be used on that usecases:
+You can use the `Edge Processor` to:
 
-* Enrich log message extra data (for example add some field or override index) using `SPL2`
-* Filter log message using `SPL2`
-* Send log messages to alternative destanations (like `AWS S3`, `Apache Kafka`, etc.)
+* Enrich log messages with extra data, such as adding a new field or overriding an index using `SPL2`.
+* Filter log messages using `SPL2`.
+* Send log messages to alternative destinations, for example, `AWS S3` or `Apache Kafka`.
```

Comment:
I think `(like `AWS S3`, `Apache Kafka`, etc.)` is better because it's available destinations of EP.
Some time later they will add few another destinations

Reply:
@ikheifets-splunk so maybe:

```suggestion
* Send log messages to alternative destinations: `AWS S3` or `Apache Kafka`.
```

what do you think?

Reply:
I don't like this one: 

`AWS S3` or `Apache Kafka`.

My proposition -  `AWS S3, Apache Kafka, etc` or `AWS S3, Apache Kafka, another available destination`

Reply:
@ikheifets-splunk the original Jennifer's update didn't limit to these choices:

```
Send log messages to alternative destinations, for example, `AWS S3` or `Apache Kafka`.
```